### PR TITLE
[bitnami/postgresql] Add option to create services for each read replica

### LINKED ISSUE #6954: [bitnam/postgresql] Allow ability to create services for each read-replica
**Which chart**:
[bitnam/postgresql]

**Is your feature request related to a problem? Please describe.**
My program manages round-robin of read-replicas by itself. Currently, being forced to use one service, I end up using only as many read-replicas as I have pods, if I get lucky and the requests go to different replicas at all.

**Describe the solution you'd like**
the ability to create services for each read-replica

**Describe alternatives you've considered**
HA-postgresql may be an alternative, but I would still encounter the situation where I dont have individual connections, relying on pg-pool configuration.

### PR DESCRIPTION
**Pull request by deefdragon**
**Description of the change**
Add option to create services for each read replica (default disabled)
Add option to create a service that connects to all read replicas. (the existing read-replica service, default enabled)

**Benefits**
Allows programs that manage connection to replicas manually to do so.
Also allows for the disabling of the service that connects to all read-replicas if not needed

**Possible drawbacks**
Read-Replica unique services do not have options for LoadBalancers, Node Ports, or Setting ClusterIP.

**Applicable issues**
  - fixes #6954

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by javsalgar** [comment]
Thank you for the PR! We will look into it

**Comment by deefdragon** [comment]
Overlooked the pattern on the values. Was not aware that was how they were updated. Just Pushed a fix. 
Might be worth denoting the proper way to update the readme on the checklist and the contribution guidelines.

File: bitnami/postgresql/templates/svc-read-set.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{{- if and .Values.replication.enabled .Values.replication.uniqueServices }}
+{{- $serviceAnnotations := coalesce .Values.readReplicas.service.annotations .Values.service.annotations -}}
+{{- $serviceType := coalesce .Values.readReplicas.service.type .Values.service.type -}}
+{{- $serviceLoadBalancerIP := coalesce .Values.readReplicas.service.loadBalancerIP .Values.service.loadBalancerIP -}}
```
Comment (javsalgar) [inline bitnami/postgresql/templates/svc-read-set.yaml:4]: Remove the variables that are not used

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you! Small change requested

File: bitnami/postgresql/templates/svc-read-set.yaml (hunk 1 above)
Comment (deefdragon) [reply bitnami/postgresql/templates/svc-read-set.yaml]: resolved

**Review by javsalgar — APPROVED** [review]
Thank you! LGTM

**Comment by deefdragon** [comment]
I committed a test file, and it looks like I need to merge in anyway. will be fixed shortly.

**Pull request merged**